docs/rst/manual/reference/configuration/keywords.rst

```diff
@@ -1784,6 +1781,13 @@ Keywords related to running the forward model
 	The QUEUE_SYSTEM keyword is optional, and usually defaults to LSF (this is
 	site dependent).
 
+.. _queue_option:
+.. topic:: QUEUE_OPTION
+
+	The chosen queue system can be configured further to for instance define the
+	resources it is using. The different queues have individual options that are
+	configurable.
+
```

Comment:
The `LSF_SERVER` option is documented without `QUEUE_OPTION` in front. Could you make it consistent with the other changes?

Reply:
I missed that one! Fixed!